v1/bundle/store.go

```diff
@@ -340,15 +353,39 @@ type ActivateOpts struct {
 	ExtraModules             map[string]*ast.Module // Optional
 	AuthorizationDecisionRef ast.Ref
 	ParserOptions            ast.ParserOptions
+	Plugin                   string
 
 	legacy bool
 }
 
+type DefaultActivator struct{}
+
+func (*DefaultActivator) Activate(opts *ActivateOpts) error {
+	opts.legacy = false
+	return activateBundles(opts)
+}
+
 // Activate the bundle(s) by loading into the given Store. This will load policies, data, and record
 // the manifest in storage. The compiler provided will have had the polices compiled on it.
 func Activate(opts *ActivateOpts) error {
```

Comment:
This is where the fallback order is decided if the caller didn't specify which bundle activation plugin to use in `bundle.ActivateOpts`.